conf/byol.yaml

```diff
@@ -0,0 +1,26 @@
+program:  # These are experiment level arguments
+  overwrite: True
+  data_dir: /datadrive/cvpr_chesapeake_landcover
```

Comment:
The default user configs shouldn't be hardcoded to only work on one person's VM

Reply:
Done